finagle-http/src/main/scala/com/twitter/finagle/http/codec/ConnectionManager.scala

```diff
@@ -12,6 +12,7 @@ private[finagle] class ConnectionManager {
   private[this] var isKeepAlive = false
   private[this] var isIdle = true
   private[this] var activeStreams = 0
+  private[this] val closeP = Promise[Unit]
```

Comment:
heh. think i copied this style from elsewhere in finagle because it surprised me. happy to standardize on new Promise ;)